test/run-xspec-tests.sh

```diff
@@ -23,8 +23,13 @@
 #       LICENSE:  MIT License
 #
 #===============================================================================
-for xspectest in *.xspec; 
-do ../bin/xspec.sh $xspectest &> result.log; 
+for xspectest in *.xspec *.schut; 
+do 
+    if test "${xspectest#.schut}" = "$xspectest"; then
```

Comment:
[Injecting `echo`](https://github.com/AirQuick/xspec/commit/4dcefff664b60b0c761375b118f2d99f5dc7ee22) for testing purposes suggests that [every file](https://travis-ci.org/AirQuick/xspec/jobs/211303084#L471) enters the Schematron route. Looks suspicious.